src/styles/header/header.scss

```
@@ -69,3 +131,40 @@
 .search-container {
   display: flex;
 }
+
+.restaurants-chefs-container {
+  display: flex;
+  gap: 32px;
+  a {
+    text-decoration: none;
+    color: $PRIMARY_BLACK;
+    font-family: Helvetica Neue;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 200;
+    line-height: normal;
+    letter-spacing: 1.92px;
+  
+    :hover{
+      text-decoration: underline;
+      text-decoration-color: rgba(222, 146, 0, 0.90);
+
+    }
+    :active {
+      text-decoration: underline;
+      text-decoration-color: rgba(222, 146, 0, 0.90);
+    }
+  }
+  @include mobile {
+    display: none;
+  }
+  @include tablet {
+    display: none;
+  }
+  @include desktop {
+    display: flex;
+  }
+  @include large-desktop {
+    display: flex;
+  }
```

you can replace all this with

```suggestion
    display: none;
  @include desktop {
    display: flex;
  }
```